Felienne/hedy
# Multiple keyword languages

This PR makes sure, English keywords are always allowed when using the hedy website in another language. So for example, when the language of the website is in Dutch, users can use English en Dutch keywords. 
To achieve this, I have updated the yaml to lark transformer in a way that English keywords are added to the lark files. 
I have also updated the keyword suggestions to suggest both Dutch keywords and English keywords. I'm not sure if this is desired though. 

Fixes issue #915 

How to test? 
Make sure local_keywords_enabled is True. 
When running hedy locally and changing the language to 'Nederlands', you should be able to use 'vraag' and 'ask'.
